orval.config.ts

```diff
@@ -2,7 +2,7 @@ import { defineConfig } from 'orval';
 
 module.exports = defineConfig({
 	base: {
-		input: './openapi.json',
+		input: 'https://api.zrosty-hay.simple2b.net/openapi.json',
```

Comment:
No, you should download the latest version of openapi.json file in your repository, and build orval from this file